torch/fx/graph.py

```diff
@@ -202,12 +211,22 @@ def _register_name_used(self, op : str) -> str:
 
     def python_code(self, root_module: str) -> str:
         free_vars: List[str] = []
-        modules_used : Set[str] = set()
+        modules_used : Set[str] = set(['typing'])
         body: List[str] = []
+
+        def type_repr(o : Any):
```

Comment:
@zdevito  I have pretty low confidence that this function is correct in all cases. Is there any generalized way you know of to go from a Python object to an expression that would represent that object?

Reply:
maybe this one would work? https://github.com/pytorch/pytorch/blob/1a2d3b6a750cc4c1709b64412619b28c737e1a79/torch/__init__.py#L231

Reply:
I don't really know. There is no guarantees here because it can always be some custom defined user class that has a weird `__repr__` defined. Or it can be a class whose name does not match the name in the global module space.  You can test it though... for instance, iterate over all of `torch` `torch.nn.function`. Look at all functions and all methods of classes. assert that `eval(repr(x)) == x` and see what breaks.